lib/modules/dosomething/dosomething_campaign/dosomething_campaign.pages.inc

```diff
@@ -31,37 +31,27 @@ function dosomething_campaign_reportback_confirmation_page($node) {
   ];
   // Image size to pass to campaign block.
   $image_size = '740x480';
-  // Signup source to pass to campaign block.
-  $source = dosomething_campaign_get_confirmation_path($node->nid);
 
-  // Set variables for anonymous user:
-  if (!user_is_logged_in()) {
```

Comment:
@DFurnes I found this gem: https://github.com/DoSomething/phoenix/blob/ec89a8484f65841358bccd50042b449ffaa34597/lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module#L404 Seems there _is_ (?) a scenario when an anonymous user would hit the confirmation page?